src/pages/Jam.svelte

```diff
@@ -55,24 +52,22 @@
     }
 
     async function getUserMedia(deviceId?: string) {
-        navigator.mediaDevices
-            .getUserMedia({
+        try {
+            mediaStream = await navigator.mediaDevices.getUserMedia({
                 audio: {
                     echoCancellation: false,
                     autoGainControl: false,
                     noiseSuppression: false,
                     deviceId,
                 },
                 video: false,
-            })
-            .then((stream) => {
-                mediaStream = stream;
-                gotStream();
-            })
-            .catch((err) => {
-                alert('getUserMedia threw exception:' + err);
-                micInit = false;
             });
+
+            gotStream();
+        } catch (error) {
+            alert('getUserMedia threw exception:' + error);
```

Comment:
I am really unsure, I will wanna check again with @pmoieni cause I dunno the plan for this one yet.

Reply:
If you think the user is supposed to know it you can use Failure here. But def check out the notiflix docs cause they have other components that are more suitable for these kind of errors instead of a toast.

Reply:
@pmoieni thanks, I will try look into what the package has to offer. will likely address it at the end before I am ready to release this draft as a PR